e2e/specs/vitals.spec.ts

```diff
@@ -45,7 +45,7 @@ test('Record vital signs', async ({ page, api }) => {
 
   await test.step('And I should see the newly recorded vital signs on the page', async () => {
     await expect(headerRow).toContainText(/temp/i);
-    await expect(headerRow).toContainText(/bp/i);
+    await expect(headerRow).toContainText(/blood pressure/i);
```

Comment:
I'm a little confused about why this works... Doesn't the header row still say BP?